ResearchKit/Common/ORKQuestionStepViewController.m

```diff
@@ -741,15 +739,6 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
             height = [ORKSurveyAnswerCellForPicker suggestedCellHeightForView:tableView];
         }
             break;
-        case ORKQuestionTypeScale:{
-            if ([self.questionStep isFormatVerticalScale]) {
-                height = [ORKSurveyAnswerCellForVerticalScale suggestedCellHeightForView:tableView];
-            }
-            else {
-                height = [ORKSurveyAnswerCellForScale suggestedCellHeightForView:tableView];
-            }
-        }
-            break;
```

Comment:
No, see comment above.  This is the removal of the hardcoded height provided by the suggestedCellHeightForView calls, which was replaced by auto layout on the cell.